### DISCUSSION TITLE
Can I add the K3S worker node to the K8S cluster？

### DISCUSSION THREAD

**Discussion by Jiaxuan-C** [k3s-io/k3s#5122] [Q&A]
Can I add the K3S worker node to the K8S cluster so that the K3S and K8S worker nodes can be managed together ？
like this：
![image](https://user-images.githubusercontent.com/88152426/153881217-9590dffd-c71a-4058-b976-70693b295d35.png)

**Comment by brandond** [comment]
K3s is K8s, but no you cannot add a K3s node to a cluster managed by a different tool (kubeadm, kubespray, minikube, etc).

**Reply by Jiaxuan-C** [reply]
thanks for your reply.
However, I want to build a cloud, edge and terminal cluster through K3S + K8S. Due to limited edge resources, I can only use the K3S work node but not the K8S work node. If the above scheme does not work, how can I achieve centralized management of the work node of K8S and K3s? Can it only be Rancher?

**Reply by brandond** [reply]
K3s **is** a K8s (Kubernetes) distribution. Is there some specific other distribution that you're trying to use it alongside? What is it that prevents you from using k3s?
